static/js/tour.js

```diff
@@ -10,6 +10,7 @@ var tour = new Tour({
     <div class='popover-content'></div> \
     <nav class='popover-navigation-rtl'> \
         <div class='btn-group' role='group'> \
+            <button class='btn btn-default' data-role='prev'><< הקודם</button> \
```

Comment:
should be with change tour1 
tour defined below...  with new function ->
 function resetTour(){
    tour = new Tour(tour1);
    tour.addStep(tour1.getStep(0));
    tour.addStep(tour1.getStep(1));
    tour.init();
}
I faced some issues with using bootstrap tour as is over the google map , 
therefore the tour is set as "array" for new steps and i use tour1 as default to push new steps and reset the tour  
that gives me the ability to navigate the tour over a spot on map